packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/leaf-more-menu.js

```diff
@@ -1,3 +1,5 @@
+// NOTE: This file is duplciated in packages/block-library/src/navigation/edit/leaf-more-menu.js
```

Comment:
This comment is not needed. They're two different things that just happen to be the same today.